support-data-rest/src/test/java/org/uniprot/api/support/data/taxonomy/controller/TaxonomySearchControllerIT.java

```diff
@@ -175,13 +222,15 @@ protected SearchParameter searchCanReturnNotFoundParameter() {
         protected SearchParameter searchAllowWildcardQueryAllDocumentsParameter() {
             return SearchParameter.builder()
                     .queryParam("query", Collections.singletonList("scientific:*"))
-                    .resultMatcher(jsonPath("$.results.*.taxonId", contains(10, 15)))
+                    .resultMatcher(jsonPath("$.results.*.taxonId", contains(10, 20, 15)))
                     .resultMatcher(
                             jsonPath(
                                     "$.results.*.scientificName",
-                                    contains("scientific10", "scientific15")))
+                                    contains("scientific10", "scientific20", "scientific15")))
```

Comment:
Inactive entry id 15 should not have scientific name

Reply:
I know, i was just reusing the existing method. We have been creating inactive entry with all these fields(e.g. scientific) set because of common method. This change just exposed what we are saving in solr.
If you want i can create a separate method to create inactive entry but i don't see much value to it as long as solr cares about active=true boost.